### PR TITLE
Daljeet/downtimes v2 main docs

### PR DESCRIPTION
**Pull request by Dalje-et**
### What does this PR do?

Explain new features and limitations that come with the new version of Downtime.s

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/monitors/downtimes/_index.md [hunk 1]
```diff
@@ -46,12 +46,49 @@ Search or use the dropdown menu to choose which monitors to silence. If the fiel
...
+| `service:web-store AND env:(dev OR staging)`       | Mutes any notification related to the `web-store` service running in either the `dev` or `staging` environment. |
+| `host:authentication-*`       | Mutes any notification that relates to a host whose name is prefixed with `authentication-`. |
+| `host:*-prod-cluster`       | Mutes any notification that relates to a host whose name is suffixed with `-prod-cluster`. |
+| `host:*-prod-cluster`       | Mutes any notification that relates to a host whose name is suffixed with `-prod-cluster`. |
+| `service:webstore AND -env:prod`       | Mutes any notification about the `web-store` service that is **not** running on the `prod` environment. |
+
+#### Downtime scope limitations
+While Downtime scope queries follow the common [Search Syntax][19], there are a few limitations that are **not supported**. These include:
+
+* More than two levels of nesting, e.g. `team:app AND (service:auth OR (service:graphics-writer AND (env:prod OR (type:metric AND status:ok))))`, are not supported. At most, Downtimes accept two levels of nesting. Use separate Downtimes instead to break down the logic.
```
Comment (estherk15) [inline content/en/monitors/downtimes/_index.md:86]: Is this a common complaint with users, not knowing search limitations? We don't typically surface what we don't support. Is this something we have plans to support in the future?

File: content/en/monitors/downtimes/_index.md [hunk 2]
```diff
@@ -164,3 +201,4 @@ Datadog can proactively mute monitors related to the manual shutdown of certain
 [16]: /integrations/amazon_ec2/#ec2-automuting
 [17]: /integrations/google_compute_engine/#gce-automuting
 [18]: /integrations/azure_vm/#automuting-monitors
+[19]: /logs/explorer/search_syntax/
```
Comment (estherk15) [inline content/en/monitors/downtimes/_index.md:204]: Would this link make more sense here or are these different?
https://docs.datadoghq.com/monitors/manage/search/

**Review by estherk15 — COMMENTED** [review]
@Dalje-et  Let me know if you have any questions on my feedback!

File: content/en/monitors/downtimes/_index.md [hunk 3]
```diff
@@ -46,12 +46,49 @@ Search or use the dropdown menu to choose which monitors to silence. If the fiel
...
+
+#### Scoping on monitor query filter
+It is common to filter a monitor query to only look at dimensions you care about. You can create Downtimes that target those same dimensions. This approach doesn't require you to add additional grouping to a monitor where it wouldn't really be necessary.
+
+{{< img src="/monitors/downtimes/downtime_scope_query.png" alt="Example of a monitor's query filter" style="width:100%;" >}}
+
+The monitor above will be muted by a Downtime that matches the monitor specific target and is scoped by `env:prod`.
+
+#### Downtime scope syntax
+The Downtime scope query follows the same common [Search Syntax][19] that many other products across the platform support. To include all groups in the scope of a Downtime, type `*` for the `Group scope`. Further examples of group scopes include:
```
[the remark this answers is not in the document]
Comment (Dalje-et) [reply content/en/monitors/downtimes/_index.md:70]: It's definitely closer to the Logs syntax (monitor search has some specific nuances that are a bit unique). I'll add more details in your other comment about the syntax!

File: content/en/monitors/downtimes/_index.md [hunk 4]
```diff
@@ -46,12 +46,49 @@ Search or use the dropdown menu to choose which monitors to silence. If the fiel
...
+| `host:authentication-*`       | Mutes any notification that relates to a host whose name is prefixed with `authentication-`. |
+| `host:*-prod-cluster`       | Mutes any notification that relates to a host whose name is suffixed with `-prod-cluster`. |
+| `host:*-prod-cluster`       | Mutes any notification that relates to a host whose name is suffixed with `-prod-cluster`. |
+| `service:webstore AND -env:prod`       | Mutes any notification about the `web-store` service that is **not** running on the `prod` environment. |
+
+#### Downtime scope limitations
+There are a few limitations that are **not supported** which include:
+
+* More than two levels of nesting, e.g. `team:app AND (service:auth OR (service:graphics-writer AND (env:prod OR (type:metric AND status:ok))))`, are not supported. At most, Downtimes accept two levels of nesting. Use separate Downtimes instead to break down the logic.
+* Negation is only supported for key/value pairs and tags with `OR`, e.g. `-key:value` and `-key(A OR B)`. Scopes such as `-service:(A AND B)`, `service:(-A OR -B)`, `-service(A B)`, etc. are not supported.
```
Comment (estherk15) [inline content/en/monitors/downtimes/_index.md:87]:
```suggestion
* Negation is only supported for key/value pairs and tags with `OR`. For example, `-key:value` and `-key(A OR B)`. Scopes such as `-service:(A AND B)`, `service:(-A OR -B)`, or `-service(A B)` are not supported.
```

File: content/en/monitors/downtimes/_index.md [hunk 5]
```diff
@@ -46,12 +46,49 @@ Search or use the dropdown menu to choose which monitors to silence. If the fiel
...
+* Top level ORs are not supported, e.g. `service:A OR host:X`. This requires two separate Downtimes.
```
Comment (estherk15) [inline content/en/monitors/downtimes/_index.md:88]:
```suggestion
* Top level ORs are not supported, for example, `service:A OR host:X`. This requires two separate Downtimes.
```

File: content/en/monitors/downtimes/_index.md [hunk 6]
```diff
@@ -46,12 +46,49 @@ Search or use the dropdown menu to choose which monitors to silence. If the fiel
...
+* Keyless tags, e.g. `prod AND service:(A or B)` or just `prod`, aren't supported. Tags need to have a key, in this case for example `env:prod`.
```
Comment (estherk15) [inline content/en/monitors/downtimes/_index.md:89]:
```suggestion
* Keyless tags, such as `prod AND service:(A or B)` or just `prod`, aren't supported. Tags need to have a key, in this case for example `env:prod`.
```

**Review by estherk15 — APPROVED** [review]

**Pull request merged**